src/pages/Home.jsx

```diff
@@ -1,43 +1,30 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
-import Button from '@material-ui/core/Button';
-import { makeStyles, Typography } from '@material-ui/core';
-import ScholarshipsMadeSimple from '../components/ScholarshipsMadeSimple';
-import HomeSection from '../components/HomeSection';
-import community from '../img/community.png';
+import React, { useEffect, useState } from 'react';
+import firebase from 'firebase';
+import { CircularProgress, makeStyles } from '@material-ui/core';
+import PublicHome from './PublicHome';
```

Comment:
Where is the PublicHome page? Did you forget to `git add` it?

Reply:
I did but it should be on the most recent commit now.